backend/battling/views.py

```diff
@@ -53,17 +55,44 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class DeleteTeam(DeleteView):
-    template_name = "battling/delete_team.html"
+class DeleteBattle(DeleteView):
+    template_name = "battling/delete_battle.html"
     success_url = reverse_lazy("home")
-    queryset = Team.objects.all()
+
+    def get_queryset(self):
+        return (
+            super()
```

Comment:
Is this working? I think it should have `Battle.objects` here, not `super().get_queryset()`

Reply:
It was working... I just used super().get_queryset() because that was how it was explained in the example I found haha But fixed it already :smile: